doc/source/changes/version_0_27.rst.inc

```diff
@@ -161,6 +164,8 @@ Miscellaneous improvements
     a   a0   a1   a2   a3
        0.0  1.0  2.0  2.5
 
+* links to documentation in the Help menu in the editor point to the installed version of larray
```

Comment:
s/links to documentation/documentation links/
s/in the editor/of the editor/?
s/to the installed version of larray/to the version of the documentation corresponding to the installed version of larray/

I am unsure of the formulation for the last item but the current version is not clear enough.